react/testing_4/src/app/screens/Home/components/FoodList/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+
+import actionCreators from "../../../../../redux/Food/actions";
+
+import Food from "./components/Food";
+
+import "./styles.css";
+
+class FoodList extends Component {
+  componentDidMount() {
+    this.props.getFoods();
+  }
+
+  renderFoods = foods => foods.map(food => <Food key={food.id} food={food} />);
```

Comment:
![But why](http://i.imgur.com/qacR5wB.gif)

Reply:
I'd move this to the render().

Reply:
I would extract it since its not related to this component. It would be a generic food renderer. I would leave component methods for implementing logic that is totally bound to the current component.
If you think that a generic renderer isn't necessary, i would move it to the render method.